drivers/afe/ad4110/ad4110.c

```diff
@@ -605,61 +605,79 @@ int32_t ad4110_setup(struct ad4110_dev **device,
 
 	/* SPI */
 	ret = spi_init(&dev->spi_dev, &init_param.spi_init);
+	if (ret)
+		goto err_init;
 
+	dev->afe_crc_en = AD4110_AFE_CRC_DISABLE;
+	dev->adc_crc_en = AD4110_ADC_CRC_DISABLE;
 	dev->addr = init_param.addr;
+	dev->data_stat = init_param.data_stat;
+	dev->data_length = init_param.data_length;
+	dev->op_mode = init_param.op_mode;
+	dev->gain = init_param.gain;
 
 	/* Device Settings */
-	ad4110_spi_do_soft_reset(dev);
+	ret = ad4110_spi_do_soft_reset(dev);
+	if (ret)
+		goto err_init;
 	mdelay(10);
 
-	if(init_param.data_stat == AD4110_ENABLE) {
-		ret |= ad4110_spi_int_reg_write_msk(dev,
-						    A4110_ADC,
-						    AD4110_REG_ADC_INTERFACE,
-						    AD4110_DATA_STAT_EN,
-						    AD4110_REG_ADC_INTERFACE_DS_MSK);
-	}
-	dev->data_stat = init_param.data_stat;
-
-	if(init_param.data_length == AD4110_DATA_WL16) {
-		ret |= ad4110_spi_int_reg_write_msk(dev,
-						    A4110_ADC,
-						    AD4110_REG_ADC_INTERFACE,
-						    AD4110_DATA_WL16,
-						    AD4110_REG_ADC_INTERFACE_WL16_MSK);
-	}
-	dev->data_length = init_param.data_length;
-
 	if(init_param.afe_crc_en != AD4110_AFE_CRC_DISABLE) {
-		ret |= ad4110_spi_int_reg_write(dev,
-						A4110_AFE,
-						AD4110_REG_AFE_CNTRL1,
-						AD4110_REG_AFE_CNTRL1_CRC_EN);
+		ret = ad4110_spi_int_reg_write(dev,
```

Comment:
In this commit you have a mix of the relevant change and removing the | from ret |= modifing also spaces.
These seems different changes and is easier to follow if seen separatly.